docs/docs/how-to/oauth.md

```diff
@@ -643,7 +643,8 @@ const secureCookie = (user) => {
     data,
     process.env.SESSION_SECRET
   ).toString()
-
+  const cookieName = 'session_8911' //if you're using dbAuth v7.6.2, you have to change the cookie name. You can comment out the line below and make the next line as comment. 8911 is the api port. You can change it accordingly.
```

Comment:
Would it be possible to import `cookieName` from `src/lib/auth`? And then pass it through `cookieName()` from `@redwoodjs/auth-dbauth-api` to do the `%port%` replacement in the cookie name string.

Reply:
I did not know that there is such a function. I'll change it.

Thank you.